unlock-app/src/components/lock/Overlay.js

```diff
@@ -54,6 +54,7 @@ export default connect(
 )(Overlay)
 
 const FullPage = styled.div`
+  @import url('https://fonts.googleapis.com/css?family=Roboto');
```

Comment:
Just a note that `themes/fonts.js` exists. If this is really the only place we're using Roboto, I'd argue that we should move to using another font we're already using. If not, we should probably bump it up and add to the main font loader? Otherwise paywall users - who are the users who will be loading our code by far most of the time - are made to download fonts from Google twice.

Reply:
cc @smombartz to weigh in on the font

Reply:
sweet I felt insecure about adding it here, I definitely think it should go in themes/fonts.js, thanks Ben!

Reply:
(pending Sascha's response)

Reply:
I went ahead and moved this into `themes/fonts.js`, we can always change the font if Sascha decides to do so later. OK for merge @benwerd ?